BuildConfigGen/Program.cs

```diff
@@ -1097,6 +1349,17 @@ private static bool ReadVersionMap(string versionMapFile, out Dictionary<string,
                     throw new Exception($"expected Default version to be present in {versionMapFile}");
                 }
 
+                if (globalVersion is not null)
+                {
+                    TaskVersion taskGlobalVersion = maxVersion.CloneWithMinorAndPatch(globalVersion.Minor, globalVersion.Patch);
+                    versionMap.Add(Config.LocalPackages.constMappingKey, taskGlobalVersion);
+
+                    if (taskGlobalVersion > maxVersion)
```

Comment:
To check: this may affect version behavior for tasks not opted-in to global version

Reply:
Replying to myself.  The behavior is preferred -- all configs should avoid conflicts with the globalversion, regardless of whether the task itself has a globalversion config.